### PR TITLE
[RF] POM> AT_12.07_001 | Freestyle project>Edit description> Verify posiblity to type the text

### PR DESCRIPTION
**Pull request by DimaCurkanu**
Add the POM-test and correct newItemPage.json 

https://trello.com/c/XE6xhmD1/1671-rf-pom-at1207001-freestyle-projectedit-description-verify-possiblity-to-type-the-text

### CODE REVIEW

File: cypress/e2e/freestyleProjectEditDescription.cy.js [hunk 1]
```diff
@@ -46,22 +46,22 @@ describe('freestyleProjectEditDescription', () => {
...
-      cy.get('.jenkins-button--primary').click();
+      // cy.get('#description-link').should('be.visible').and('includes.text', createNewView.description).click();
+      // cy.get('.jenkins-input').should('be.visible').type(createNewView.typeText)
+      // cy.get('.jenkins-button--primary').click();
+      // cy.get('#description').contains(createNewView.typeText);
+      // cy.get('#description-link').should('be.visible').and('includes.text', createNewView.description).click();
+      // cy.get('.jenkins-input').clear();
+      // cy.get('.jenkins-input').should('be.visible').type(createNewView.editDescriptionText);
+      // cy.get('.jenkins-button--primary').click();
       cy.get('#description').contains(createNewView.editDescriptionText);
```
Comment (LanaBondarenko) [inline cypress/e2e/freestyleProjectEditDescription.cy.js:65]: We can't to use comments in our files.

File: cypress/e2e/tests/freestyleProject.cy.js [hunk 2]
```diff
@@ -10,18 +10,23 @@ import { errorMessage } from "../../fixtures/pom_fixtures/freestyleProjectPage.j
 import freestyleConfigurePage from "../../fixtures/pom_fixtures/freestyleConfigurePage.json"
 import freestyleProject from"../../fixtures/pom_fixtures/freestyleProjectPage.json"
 import FreestyleProjectPage from "../../pageObjects/FreestyleProjectPage";
+import NewItemPage from "../../pageObjects/NewItemPage";
+import newItemPage from "../../fixtures/pom_fixtures/newItemPage.json";
```
Comment (LanaBondarenko) [inline cypress/e2e/tests/freestyleProject.cy.js:14]: As was mentioned during the lecture when we use data from fixture file we have to add "Data" at the and of Variable name.

File: cypress/fixtures/pom_fixtures/newItemPage.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "multibranchPipelineName": "Multibranch Pipeline Job",
-    "freestyleProjectName": "Freestyle Project Job",
+    "newItemPage": "Freestyle Project Job",
     "orgFolderName": "OrgFolderTest",
```
Comment (LanaBondarenko) [inline cypress/fixtures/pom_fixtures/newItemPage.json:4]: This name does not reflect the value of the variable. Different types of jobs are created on this page, and to emphasize which job is being created, this is indicated in the variable name.

**Review by EkBlinova — DISMISSED** [review]

**Review by Rom-T — APPROVED** [review]
Good job!

**Pull request merged**